client/logic/game/units/unit.gd

```diff
@@ -258,15 +258,14 @@ func play_attack(weapon_category: GameConst.WeaponCategory) -> void:
 ## Function for damage animated signal.
 ## [br]
 ## The effect is placed in a randomly position
-## within a distance of [param radius] from the unit.
-func play_damage(radius: int = 0) -> void:
+## within a distance of [param box_radius] from the unit.
+func play_damage(box_radius: Vector2 = Vector2.ZERO) -> void:
```

Comment:
more that box_radius should be box_size, right? radius is only for circles, and box, I expect something square (at least my understanding)

Reply:
Yeah, I struggled to come up with a better name for it, as my initial idea was to refer to the shape as an 'oval area'.